rll/metadata.rb

```diff
@@ -58,3 +51,9 @@
   :default => "cred:AWS_ACCESS_KEY_ID",
   :recipes => ["rll::test-script"]
 
+attribute   "DISABLE_AUTO_UPGRADE",
+  :display_name => "Disables auto upgrade if previously enabled",
```

Comment:
What do you mean by "if previously enabled"? Isn't it enabled by default and this disables it? Why not create an "auto_upgrade" variable and set it to true by default?